docs/src/drivers/gm.adoc

```diff
@@ -331,20 +326,20 @@ image::images/GM_RefSignals.png["Reference signal timing diagrams",align="center
 
 Setting StepGen module of axis 0 to generate 1000 step pulse per position unit:
 
-----
-setp gm.0.stepgen.0.step-type 0         # 0:stepDir, 1:UpDown, 2:Quad
-setp gm.0.stepgen.0.control-type 0      # 0:Pos. control, 1:Vel. Control
-setp gm.0.stepgen.0.invert-step1 0
-setp gm.0.stepgen.0.invert-step2 0
-setp gm.0.stepgen.0.maxvel 0            # do not set maxvel for step
-                                        # generator, let interpolator control it.
-setp gm.0.stepgen.0.maxaccel 0          # do not set max acceleration for
-                                        # step generator, let interpolator control it.
-setp gm.0.stepgen.0.position-scale 1000 # 1000 step/position unit
-setp gm.0.stepgen.0.steplen 1000        # 1000 ns = 1 µs
-setp gm.0.stepgen.0.stepspace1000       # 1000 ns = 1 µs
-setp gm.0.stepgen.0.dirdelay 2000       # 2000 ns = 2 µs
-----
+[options="header"]
+|===
+| setp pin seetting                  | Comment
+l|setp gm.0.stepgen.0.step-type 0    | 0:stepDir, 1:UpDown, 2:Quad
+l|setp gm.0.stepgen.0.control-type 0 | 0:Pos. control, 1:Vel. Control
+l|setp gm.0.stepgen.0.invert-step1 0 |
+l|setp gm.0.stepgen.0.invert-step2 0 |
+l|setp gm.0.stepgen.0.maxvel 0       |do not set maxvel for step lenerator, let interpolator control it.
+l|setp gm.0.stepgen.0.maxaccel 0     |do not set max acceleration for step generator, let interpolator control it.
+l|setp gm.0.stepgen.0.position-scale 1000 |1000 step/position unit
+l|setp gm.0.stepgen.0.steplen 1000   | 1000 ns = 1 µs
+l|setp gm.0.stepgen.0.stepspace 1000 | 1000 ns = 1 µs
+l|setp gm.0.stepgen.0.dirdelay 2000  | 2000 ns = 2 µs
+|===
```

Comment:
Formatting this as table looses the connection that it is a code example. I would rather add `[source,{hal}]` to the code block.

Reply:
Hm. Only the left side is executable, not the comment on the right.

Reply:
>Agree with @hansu so same comment and proposal as above.
"Right side" could be converted to # <n> callouts but not convinced it would be beneficial here.

👍